src/main/java/org/jabref/gui/desktop/os/Linux.java

```diff
@@ -20,23 +21,39 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Linux.class);
 
+    private void nativeOpenFile(String filePath) {
+        if (Desktop.isDesktopSupported()) {
+            new Thread(() -> {
+                try {
+                    File file = new File(filePath);
+                    Desktop.getDesktop().open(file);
+                    LOGGER.info("Opened file with Desktop integration");
+                } catch (Exception e) {
+                    LOGGER.error("Open operation not successful: " + e.getMessage());
```

Comment:
```suggestion
                    LOGGER.error("Open operation not successful", e);
```
Logger error has an overload which directly accepts an throwable/eception